docs/tutorial.md

```diff
@@ -344,6 +344,45 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace.
 
+## SRE Recipes
+
+SRE Recipes is our [Chaos Engineering](https://en.wikipedia.org/wiki/Chaos_engineering) tool to test your sandbox environment. It helps users to familiarize with finding the root cause of a breakage using Cloud Ops monitoring.  
+Each 'recipe' simulate a different scenario, there are several recipes that you can run and you can also [contribute your own.](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/sre-recipes#contributing)  
```

Comment:
Maybe expand a little bit: e.g. "each recipe simulates a different scenario of real life problems that can occur to production system.